### ISSUE TITLE
I found some bug

### ISSUE THREAD

**Issue by mikhail2013ru** [SWivid/F5-TTS#1075]
### Checks

- [x] This template is only for bug reports, usage problems go with 'Help Wanted'.
- [x] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [x] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [x] I am using English to submit this issue to facilitate community communication.

### Environment Details

Hello. 
1. Not every clone record fragment generation ends with successful generation of a new record. I have already had 2 times when I waited for more than 100 steps and after the last steps the record remained old.
2. the inability to stop the generation, for example, if you notice errors in the text. It is necessary to add a stop synthesis button, this is especially important on colab.
3. When replacing the clone sample record, phrases from the old clone sample are found in the generated record, this is solved only by reloading the page.
4. Why is it not possible to synthesize a voice directly from a voice model, and not just from a cloned fragment of speech?

### Steps to Reproduce

1. Not every clone record fragment generation ends with successful generation of a new record. I have already had 2 times when I waited for more than 100 steps and after the last steps the record remained old.
2. the inability to stop the generation, for example, if you notice errors in the text. It is necessary to add a stop synthesis button, this is especially important on colab.
3. When replacing the clone sample record, phrases from the old clone sample are found in the generated record, this is solved only by reloading the page.
4. Why is it not possible to synthesize a voice directly from a voice model, and not just from a cloned fragment of speech?

**Comment by SWivid** [comment]
Hi @mikhail2013ru , maybe you can help confirm if I get you right first:

1. Not quite get this. You mean, multiple time synthesis (random seed not fixed) but get all same result, or just part of them? Maybe some example to help us get this bug behavior.
2. Need a stop button to interrupt generation process.
3. Replace the previous prompt wav with a new one, but the corresponding transcription (i.e. text) not changed (in basic_tts feature?)
4. To synthesize without providing a prompt wav.

**Comment by mikhail2013ru** [comment]
I'll try to explain it differently: 
1. I use the same 2,000-character text for synthesis, but I upload different sections of recordings one at a time for cloning. If you do not refresh the page and reload the segment, phrases from the old recording appear in the recorded speech. I've noticed this more than once. 
2. The optimal duration of the reference sample for cloning is not clear. Do you need 1, 2, 3 or more minutes for the best cloning? 
3. The stop synthesis button is necessary, especially on colab. 
4.  Make a generation from a voice model, not just from a cloned sample.
5.  Synthesis sometimes breaks down after completing all the stages, there is no new recording, only the old one is available. I came across this today after 100 steps.
6.  Create a script to retrain the finetune model from a checkpoint in the web interface, stopping and resuming training for a new voice.
7.  Add the audio file export to .mp3 format

**Comment by SWivid** [comment]
> phrases from the old recording appear in the recorded speech

That's what makes me confused. Let us use the same term, for example, just take the name of things from the UI interface.
Do you mean `Reference Text` by `phrases`, and `Reference Audio` by `old recording` , `Synthesized Audio` by `the recorded speech`?

And just make sure we are talking about the same thing: are you using `infer-cli` or `infer-gradio`; if `infer-gradio`, which feature did you use: `Basic-TTS`, `Multi-Speech`, or `Voice-Chat`?
The issue template has mentioned that would need these things for better bug tracing.

**Comment by mikhail2013ru** [comment]
> > Фразы из старой записи появляются в записанной речи
> 
> Вот что меня сбивает с толку. Давайте использовать один и тот же термин, например, просто возьмём названия элементов из пользовательского интерфейса. Вы имеете в виду `Reference Text` под `phrases`, и `Reference Audio` под `old recording`, `Synthesized Audio` под `the recorded speech`?
> 
> И просто убедитесь, что мы говорим об одном и том же: вы используете `infer-cli` или `infer-gradio`; если `infer-gradio`, то какую функцию вы использовали: `Basic-TTS`, `Multi-Speech`, или `Voice-Chat`? В шаблоне проблемы указано, что для более эффективной отладки ошибок необходимы эти элементы.

Sorry to mislead you, I'll clarify in more detail. I do record synthesis in the Basic-TTS tab. In Reference Audio, I take turns downloading mp3 files with a duration of 1 minute. I leave the Reference Text empty. Since the first launch of the virtual environment on colab, I have been trying out a new reference voice sample and voicing 1.5-2 thousand characters. If the speech is acceptable, I will voice the texts up to 15-20 thousand characters at a time. Next, in Reference Audio, I change the voice sample, change the Reference Text, and on the new recording I hear phrases from the previous Reference Audio sample. I also tried to vocalize more than 30 thousand characters, and after finishing all the steps, I did not receive a new entry. I repeat, the "Stop synthesis" button is needed on colab, in case the generation needs to be interrupted and edits made to the text. Otherwise, the virtual environment will have to be deleted and a new one created, otherwise I will be in the generation queue. I run the web interface via gradio.

**Comment by SWivid** [comment]
> Next, in Reference Audio, I change the voice sample, change the Reference Text, and on the new recording I hear phrases from the previous Reference Audio sample.

I think you have not changed accordingly the Reference Text, it is in the Adanced Setting.
But the current version will do auto-clear-up if Reference Audio is removed.

> I repeat, the "Stop synthesis" button is needed on colab, in case the generation needs to be interrupted and edits made to the text. Otherwise, the virtual environment will have to be deleted and a new one created, otherwise I will be in the generation queue. I run the web interface via gradio.

Why you need to delete virtual environment? I think you can click stop button from colab.

---
> 2\. The optimal duration of the reference sample for cloning is not clear. Do you need 1, 2, 3 or more minutes for the best cloning?
> ...
> 6\. Create a script to retrain the finetune model from a checkpoint in the web interface, stopping and resuming training for a new 
voice.
> ...

For other questions, I feel like you have never read the documentation of this repository. For example, as you said you are using infer-gradio, there is already clear instruction for Reference Audio length both in README.md and in gradio UI interface.
Samely for training scripts.
